.ruby-version

```diff
@@ -1 +1 @@
-2.3.0
+2.3.4
```

Comment:
Nope no need 😄 It was just my curiosity 👍 

My guess is that it won't affect the end users of the gem too 😉